packages/jest-message-util/src/index.js

```diff
@@ -194,15 +196,45 @@ export const formatStackTrace = (
   testPath: ?Path,
 ) => {
   let lines = stack.split(/\n/);
+  let renderedCallsite = '';
   const relativeTestPath = testPath
     ? slash(path.relative(config.rootDir, testPath))
     : null;
   lines = removeInternalStackEntries(lines, options);
-  return lines
+
+  if (testPath) {
+    const topFrame = lines
+      .join('\n')
+      .trim()
+      .split('\n')[0];
+
+    const parsedFrame = StackUtils.parseLine(topFrame);
+
+    if (parsedFrame) {
+      renderedCallsite = codeFrameColumns(
+        fs.readFileSync(testPath, 'utf8'),
+        {
+          start: {line: parsedFrame.line},
+        },
+        {highlightCode: true},
+      );
+
+      renderedCallsite = renderedCallsite
+        .split('\n')
+        .map(line => MESSAGE_INDENT + line)
+        .join('\n');
+
+      renderedCallsite = `\n\n${renderedCallsite}\n\n`;
```

Comment:
rm one before and after plsss

Reply:
done.

![image](https://user-images.githubusercontent.com/1404810/34038640-512bfab2-e18d-11e7-8fc6-1fdbaa0b981a.png)